content/en/docs/tasks/configure-pod-container/image-volumes.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Use an Image Volume With a Pod
+reviewers:
+content_type: task
+weight: 210
+min-kubernetes-server-version: v1.31
+---
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="ImageVolume" >}}
+
+This page shows how to configure a pod using image volumes. This allows you to
+mount content from OCI registries inside containers.
+
+The feature has been developed as part of [Kubernetes enhancement #4639](https://kep.k8s.io/4639).
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% thirdparty-content single="true" %}}
+
+- The node OS needs to be Linux
+- The container runtime needs to support the image volumes feature.
```

Comment:
```suggestion
- The container runtime needs to support the image volumes feature
```

To dot or not to dot, that is a question. :smile:

Reply:
Yep, removed the dot.